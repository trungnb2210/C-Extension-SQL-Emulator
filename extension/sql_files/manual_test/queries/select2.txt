SELECT object, quantity
FROM prices