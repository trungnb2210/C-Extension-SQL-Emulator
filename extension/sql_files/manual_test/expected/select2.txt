QUERY RESULT:
  object   quantity 
---------------------
| Box    | 2        | 
| Box    | 7        | 
| Tire   | 1        | 
| Table  | 5        | 
| Pen    | 50       | 
---------------------
